app/build/app/js/app.js

```diff
@@ -2,7 +2,7 @@ angular.module('media_manager', ['ui.bootstrap', 'ngRoute', 'ngDroplet', 'xedita
 .config(['$routeProvider', function($routeProvider){
   $routeProvider
   .when('/', {
-    redirectTo: "/collections"
+    redirectTo: "/workspace"
```

Comment:
Shouldn't this be redirecting to the list view? Not sure why this was changed to redirect to the workspace.